dev-docs/bidders/adnuntius.md

```diff
@@ -2,193 +2,31 @@
 layout: bidder
 title: Adnuntius
 description: Prebid Adnuntius Bidder Adaptor
-pbjs: true
-pbs: true
 biddercode: adnuntius
-media_types: banner, video
+pbjs: true
+media_types: banner, video, native
 tcfeu_supported: true
-fpd_supported: true
-gvl_id: 855
-safeframes_ok: false
-multiformat_supported: will-bid-on-any
```

Comment:
Why did you remove `multiformat_supported` ? Please add again

Reply:
Thank you for saving me from myself!

Updated the wrong files because analytics/adnuntius.md and bidders/adnuntius.md can look very similar in an IDE!

All updated now.